src/managers/GuildMemberManager.js

```diff
@@ -135,6 +136,48 @@ class GuildMemberManager extends BaseManager {
     return this._fetchMany(options);
   }
 
+  /**
+   * Edits a member of the guild.
+   * <warn>The user must be a member of the guild</warn>
```

Comment:
I don't think this needs to be a warn, I think an info should be more than enough